AutomatticTracks/build.gradle

```diff
@@ -7,6 +7,13 @@ plugins {
 repositories {
     google()
     mavenCentral()
+    maven {
+        url "https://a8c-libs.s3.amazonaws.com/android"
+        content {
+            includeGroup "org.wordpress"
+            includeGroup "org.wordpress.lint"
```

Comment:
```suggestion
```

The `group` for lint library is `org.wordpress` - there's no need to add the other group (which is not valid AFAIK, `lint` is the name of a module)

Reply:
Yeap, I wasn't 100% sure about adding `org.wordpress.lint` too, I just followed the same pattern across all of our other repos, see this [Login](https://github.com/rohankumardubey/WordPress-Login-Flow-Android/blob/3ab114c06b2311e4f25d28adced9f70c34bfbbac/WordPressLoginFlow/build.gradle#L27-L33) repo as an example... 🤷 

Wdyt @wzieba ? 🤔

Reply:
Also to make the `example` module work on the `Catalog` repo, I added [this](https://github.com/Automattic/android-dependency-catalog/blob/trunk/example/settings.gradle.kts#L36) line there too, following the [Adding a new dependency](https://github.com/Automattic/android-dependency-catalog/blob/trunk/example/settings.gradle.kts#L36) documentation, as I was indeed hitting a problem on it... 🤷 

> [Build the catalog & example projects](https://github.com/Automattic/android-dependency-catalog#building-locally) - if you get Could not find {dependency} error, make sure the repository for the new dependency you've added is included in [example/settings.gradle.kts](https://github.com/Automattic/android-dependency-catalog/blob/trunk/example/settings.gradle.kts)

Reply:
> Yeap, I wasn't 100% sure about adding org.wordpress.lint too, I just followed the same pattern across all of our other repos, see this Login repo as an example... 🤷

It seems that this part in `Login` is also incorrect - it adds a valid group `org.wordpress` but also invalid and unnecessary one: `org.wordpress.fluxc`.

> Also to make the example module work on the Catalog repo, I added this line there too, following the Adding a new dependency documentation, as I was indeed hitting a problem on it... 🤷

Yes, this line is correct - it adds a new `module` (not `group`) so it specifies `lint` module with it's group: `org.wordpress`.

Reply:
Got it, thanks for the explanation @wzieba ! 🥇 

Done: 1e055a63d5253553bd160f3998736df642640512